run.sh

```diff
@@ -7,12 +7,15 @@ function log {
     echo "---------------------------------------"
 }
 
-log "Get Weather info"
+log "Add weather info"
 python3 screen-weather-get.py
 
-log "Get Calendar info"
+log "Add Calendar info"
 python3 screen-calendar-get.py
 
+log "Add Custom data"
+python3 screen-custom-get.py
```

Comment:
By default, ends up adding an empty string to the output, nothing visible.  But a user can tinker with the SVG and modify it and show extra things.